test/HealthChecks.AzureStorage.Tests/DependencyInjection/AzureFileShareStorageRegistrationTests.cs

```diff
@@ -7,7 +7,7 @@ public void add_health_check_when_properly_configured()
         {
             var services = new ServiceCollection();
             services.AddHealthChecks()
-                .AddAzureFileShare("the-connection-string");
+                .AddAzureFileShare("FileEndpoint=https://unit-test.file.core.windows.net");
```

Comment:
Is such change nessessary for test purposes? I don't mind to change fake connection string to some real value but I am interested in whether this change is caused by an earlier creation of the client and connection string parsing.

Reply:
You're right -- it's because we create the client in the ctor now instead of within the `CheckHealthAsync` method. The underlying service client parses the connection string and will throw if it's not formatted properly. We still don't need the corresponding service to exist, but the value must parse correctly.

Reply:
ok